source/scenes/common/Login/Login.tsx

```diff
@@ -114,17 +120,16 @@ const Login = ({
             placeholder={PLEASE_ENTER_YOUR_PASSWORD_STRING}
             variant={styles.password}
           />
-          <div className={styles.errorWrapper}>
-            {errors.password ? (
-              <span className={errorClass}>{errors.password.message}</span>
+          <div id={'ui-login-status'} className={styles.errorWrapper}>
+            {isInvalid ? (
+              <span id={'ui-login-error'} className={errorClass}>{failureComment}</span>
             ) : (
-              isInvalid && (
-                <span className={errorClass}>Error: Invalid password</span>
-              )
+              <span id={'ui-login-success'} className={errorClass}>{successComment}</span>
```

Comment:
This shows all the time - even before any action is taken. Let's remove this message completely.
![Screen Shot 2022-01-11 at 10 59 49 AM](https://user-images.githubusercontent.com/3005444/149004894-db643bb3-420f-4023-8875-c23b234f5256.png)

Reply:
removed the state "Login successful." statement